.github/actions/spelling/line_forbidden.patterns

```diff
@@ -1,119 +1,842 @@
+\b[Cc]an not\b(?! only\b)
+
+# Should be `chart`
+(?i)\bhelm\b.*\bchard\b
+
+# Should be `code`
+(?<=\bof )a code(?= that\b)
+
+# Should be `counter-intuitive`
+\bcounter intuitive\b
+
+# Do not use `(click) here` links
+# For more information, see:
+# * https://www.w3.org/QA/Tips/noClickHere
+# * https://webaim.org/techniques/hypertext/link_text
+# * https://granicus.com/blog/why-click-here-links-are-bad/
+# * https://heyoka.medium.com/dont-use-click-here-f32f445d1021
+(?i)(?:>|\[)(?:(?:click |)here|link|(?:read |)more)(?:</|\]\()
 
-# s.b. GitLab
-(?<![&*.]|// |\btype )\bGitlab\b(?![{)])
+# Including "image of" or "picture of" in alt text is unnecessary.
+\balt=['"](?:an? |)(?:image|picture) of
 
-# s.b. JavaScript
+# Alt text should be short
+\balt=(?:'[^']{126,}'|"[^"]{126,}")
+
+# Should be either of `default` or `fallback`, but not both
+# Unless you have a non-default fallback, but that's just weird.
+\bdefault fallback\b
+
+# Should be `dress code`
+\bdressing code\b
+
+# Should be `effect`
+(?<=\btake )affect\b
+
+# Should be `end`
+(?<=\b[Ww]e )ends\b
+
+# Should be `ends`
+\bend's(?= up\b)
+
+# Should be `-endian`
+\b(?i)(?<=big|little) endian\b
+
+# Should be `equals` to `is equal to`
+\bequals to\b
+
+# Should be `equal to`, `equal width`, or reworded
+\bequal with\b(?! (?:no|respect))
+
+# Should be `eta` or `calculate eta`
+(?i)\bestimated? eta\b
+
+# Should be `ECMA` 262 (JavaScript)
+(?i)\bTS\/EMCA\b|\bEMCA(?: \d|\s*Script)|\bEMCA\b(?=.*\bTS\b)
+
+# Should be `ECMA` 340 (Near Field Communications)
+(?i)EMCA[- ]340
+
+# Should be `exceed`
+\bare higher than\b
+
+# Should be `exceeds`
+\bis higher than\b
+
+# Should be `fall back`
+(?<!\ba )\bfallback(?= to)\b
+
+# Should be `for`, `for, to` or `to`
+\b(?:for to|(?<!\bup )to for)\b
+
+# Should be `ghcr.io`
+# https://bmitch.net/blog/2025-08-22-ghrc-appears-malicious/
+\bghrc\.io\b
+
+# Should be `GitHub`
+(?<![&*.]|// |\b(?:from|import|type) )\bGithub\b(?![{()])
+
+# Should be `GitLab`
+(?<![&*.]|// |\b(?:from|import|type) )\bGitlab\b(?![{()])
+
+# Should be `GmbH`
+\bGmbh\b
+
+# Should be `heartrending` unless talking about drawing hearts
+\b(?i)heart[- ]rendering\b(?![^.?!]*(?:hearts|quirk))
+
+# Should probably be `https://`...
+# Markdown generally doesn't assume that links are to urls
+\]\(www\.\w
+
+# Should be `https://www.chiark.greenend.org.uk/~sgtatham/putty/`
+# See https://hachyderm.io/@simontatham/114846017785770922
+\bputty\.org\b
+
+# Often should be `if`/`when` (or reworded)
+# unless describing exceptional cases (i.e. emergencies)
+#(?<!just )\bin case\b(?! (?:I|s?he|they|of|we|you))
+
+# Should be `in`
+#(?<=\b(?:go|reach) back )into(?= time\b)
+
+# Should be `in` or `out`
+(?<=\bfill )up(?= the forms?\b)
+
+# Should be `intents and purposes`
+(?<=[Ff]or all )intensive purposes\b
+
+# Should be `JavaScript`
 \bJavascript\b
 
-# s.b. macOS or Mac OS X or ...
+# Should be `Jira`
+\bJIRA\b
```

Comment:
Unless you're tracking things you are doing to manage Jira within itself. This should be `\bJIRA\b(?!-\d+)` or something along those lines, to avoid flagging `JIRA-12345`.

Reply:
Thanks. I've fixed it in https://github.com/check-spelling/spell-check-this/commit/9d030417e43386d6b6d0fe59585bde53c5bfe304 for the baseline.